components/post_view/post_list.jsx

```diff
@@ -452,7 +453,9 @@ export default class PostList extends React.PureComponent {
                 });
             }
 
-            if (postList.scrollTop <= 0.5 * postList.clientHeight && !this.state.loadingPosts && !this.state.atEnd && this.state.autoRetryEnable) {
+            const didScrollPastLoadingPoint = postListScrollTop < 0.3 * (postList.scrollHeight - postList.clientHeight);
+
+            if ((didScrollPastLoadingPoint || postListScrollTop < 300) && !this.state.loadingPosts && !this.state.atEnd && this.state.autoRetryEnable) {
```

Comment:
Can we make `0.3` and `300` constants so we can easily adjust it in case we need in the future?  Thanks

Reply:
Yup. Sure. Added constants. As we talked yesterday Added a max height check  as well after talking to eric